rust/xaynet-analytics/Cargo.toml

```diff
@@ -19,4 +19,4 @@ rustdoc-args = ["--cfg", "docsrs"]
 [dependencies]
 anyhow = "1.0.38"
 chrono = "0.4.19"
-isar-core = { git = "https://github.com/isar/isar-core", rev = "9165741a3660c84747596d2dd5368cd13c1e9171" }
+isar-core = { git = "https://github.com/isar/isar-core", rev = "59d9008be33343d1fd313c659e50e2835365a19d" }
```

Comment:
Why do we need to update isar?

Reply:
He improved the schema registration and i think he solved some bug with it. See the changes in `SchemaGenerator` in `common.rs`. I was having troubles with the Isar properties, so i thought i'd update. Honestly i don't think he's done changing stuff in the public API, maybe it makes sense to keep updating. But will depend on your priorities.

Reply:
ah forgot! he also removed the need for the [ordering of properties registration](https://github.com/xaynetwork/xaynet/blob/530e01e837e0e2176134a22cb098b7ac078b7110/rust/xaynet-analytics/src/database/analytics_event/adapter.rs#L41) that you were concerned about. i'll update those methods